docs/user-guide/ze-profiles.md

```diff
@@ -1,22 +1,18 @@
-# Zowe Explorer profiles
+# Creating Zowe Explorer profiles
 
 After you install Zowe Explorer, you must have a Zowe Explorer profile to use all functions of the extension.
-
-:::info
-You can continue using Zowe V1 profiles with Zowe Explorer V2. See [Working with Zowe CLI V1 profiles](#working-with-zowe-cli-v1-profiles) for more information.
+:::info Required roles: Systems administrator
 :::
 
-## Configuring Zowe V2 profiles
-
-Zowe V2 uses *team profiles* to simplify profile management by letting you edit, store, and share mainframe connection details in one location, a configuration file.
+## Configuring Zowe profiles
 
-You can use a text editor or an IDE to populate configuration files with the connection information for your mainframe services. By default, your *global* team configuration file is located in the `.zowe` home folder, whereas the *project* configuration file is located in the main directory of your project.
+Zowe uses [team configuration](/appendix/zowe-glossary.md#team-configuration) to simplify profile management by letting you edit, store, and share mainframe connection details in one location, a configuration file.
 
-You can create profiles that you use globally, given that the names of the globally-used profiles are different from your other profile names.
+You can use a text editor or an IDE to populate configuration files with profiles, which contain the connection information for your mainframe services. By default, your *global* team configuration file is located in the `.zowe` home folder, whereas a *project* configuration file is located in the main directory of your project.
```

Comment:
Regarding:
> By default, your *global* team configuration file is located in the `.zowe` home folder.

I think we should adjust the wording to clarify that the name of the folder might not be `.zowe`, depending on how the environment is configured. Zowe client components first look for the `ZOWE_CLI_HOME` environment variable, and if it exists, that path is used to store the global team config, internal Zowe settings, etc. If the variable is not set, then it creates and uses a `.zowe` folder in the user's home directory.